spine-ts/spine-pixi-v8/src/Spine.ts

```diff
@@ -359,12 +368,72 @@ export class Spine extends ViewContainer {
 		this.spineAttachmentsDirty = spineAttachmentsDirty;
 	}
 
+	private updateAndSetPixiMask (slot: Slot, last: boolean) {
+		// assign/create the currentClippingSlot
+		const attachment = slot.attachment;
+		if (attachment && attachment instanceof ClippingAttachment) {
+			const clip = (this.clippingSlotToPixiMasks[slot.data.name] ||= { slot, vertices: new Array<number>() });
+			clip.maskComputed = false;
+			this.currentClippingSlot = this.clippingSlotToPixiMasks[slot.data.name];
+			return;
+		}
+
+		// assign the currentClippingSlot mask to the slot object
+		let currentClippingSlot = this.currentClippingSlot;
+		let slotObject = this._slotsObject[slot.data.name];
+		if (currentClippingSlot && slotObject) {
+			let slotClipping = currentClippingSlot.slot;
+			let clippingAttachment = slotClipping.attachment as ClippingAttachment;
+
+			// create the pixi mask, only the first time and if the clipped slot is the first one clipped by this currentClippingSlot
+			let mask = currentClippingSlot.mask as Graphics;
+			if (!mask) {
+				mask = new Graphics();
```

Comment:
this is all good! one suggestion would be to pull Graphics from a pool, rather than create a new one each time.
This depends on how frequently the masks are added and removed though, so might be cool as is!

Reply:
Good suggestion and thanks for the review 👍 

If a user switches between two animations frequently, and one of them has slot objects affected by a clipping attachment, I'd say that it might be quite frequent.
However, I expect slot objects being used rarely...but we know our users.
So, I suppose it's worth to create a pool for these masks. I'll work on that tomorrow.